holidays/locale/ar/LC_MESSAGES/UA.po

```diff
@@ -3,95 +3,101 @@
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: Python Holidays 0.26\n"
-"POT-Creation-Date: 2023-04-12 11:32-0700\n"
-"PO-Revision-Date: 2023-05-23 09:21-0700\n"
-"Last-Translator: Arkadii Yakovets <ark@cho.red>\n"
+"Project-Id-Version: Python Holidays 0.29\n"
+"POT-Creation-Date: 2023-07-08 19:40+0300\n"
+"PO-Revision-Date: 2023-07-09 17:35+0300\n"
+"Last-Translator: ~Jhellico <jhellico@gmail.com>\n"
 "Language-Team: Python Holidays localization team\n"
 "Language: ar\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=6; plural=(n==0 ? 0 : n==1 ? 1 : n==2 ? 2 : n%100>=3 && n%100<=10 ? 3 : n%100>=11 && n%100<=99 ? 4 : 5);\n"
 "Generated-By: Lingua 4.15.0\n"
-"X-Generator: Poedit 3.3.1\n"
+"X-Generator: Poedit 3.2.2\n"
+
+#. Day off (substituted from {day}.{month}).
+#: ./holidays/countries/ukraine.py:74
+#, python-format
+msgid "Вихідний день (перенесено з {day:02}.{month:02})"
+msgstr "يوم عطلة (استبدل من {day:02}/{month:02})"
```

Comment:
@abh31000 I am not sure if this translation and the chosen date format are correct, so I would be happy to hear your opinion.

Reply:
```suggestion
msgstr "يوم عطلة (استبدل من {يوم:02}.{شهر:02})"
```
Yeah like this it's correct. I also think "day" and "month" should be translated

Reply:
No, here "day" and "month" is variable substitutions for` str.format()`:
```python
self._add_holiday(
    self.tr(substituted_label).format(day=from_day, month=from_month), to_month, to_day
)
```
And it looks like I'm going to replace it. :)

Reply:
Ah my bad ! :sleeping: 
Both the translation and date format are correct.

Reply:
Now date format and substituted holiday name is set separately.